File: swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/IO/Swagger/Model/RawMail.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Raw Email Object
  /// </summary>
  [DataContract]
  public class RawMail {
    /// <summary>
    /// The entire email contents
    /// </summary>
    /// <value>The entire email contents</value>
    [DataMember(Name="raw_email", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "raw_email")]
    public string RawEmail { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class RawMail {\n");
      sb.Append("  RawEmail: ").Append(RawEmail).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: swagger-client/dart/lib/model/raw_mail.dart
part of swagger.api;

class RawMail {
  /* The entire email contents */
  String rawEmail = null;

  RawMail();

  @override
  String toString() {
    return 'RawMail[rawEmail=$rawEmail, ]';
  }

  RawMail.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    rawEmail = json['raw_email'];
  }

  Map<String, dynamic> toJson() {
    return {
      'raw_email': rawEmail
     };
  }

  static List<RawMail> listFromJson(List<dynamic> json) {
    return json == null ? new List<RawMail>() : json.map((value) => new RawMail.fromJson(value)).toList();
  }

  static Map<String, RawMail> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, RawMail>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new RawMail.fromJson(value));
    }
    return map;
  }
}
